apps/journeys-admin/src/components/ContainedIconButton/ContainedIconButton.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import { Meta, Story } from '@storybook/react'
+import Paper from '@mui/material/Paper'
+import { ComponentProps } from 'react'
+import Stack from '@mui/material/Stack'
+import NoteAddIcon from '@mui/icons-material/NoteAdd'
+import AddIcon from '@mui/icons-material/Add'
+import EditIcon from '@mui/icons-material/Edit'
+import { noop } from 'lodash'
+import { simpleComponentConfig } from '../../libs/storybook'
+import { ContainedIconButton } from '.'
+
+const ContainedIconButtonDemo = {
+  ...simpleComponentConfig,
+  component: ContainedIconButton,
+  title: 'Journeys-Admin/ContainedIconButton'
+}
+
+const Template: Story<ComponentProps<typeof ContainedIconButton>> = ({
+  ...args
+}) => (
+  <Paper elevation={0} sx={{ p: 2 }}>
+    <Stack direction="row">
+      <ContainedIconButton {...args} />
+    </Stack>
+  </Paper>
```

Comment:
We usually use paper for stories that need the background/ needs it to provide context for multiple components.
```suggestion
  <Box sx={{maxWidth: '300px'}}>
      <ContainedIconButton {...args} />
    </Box>
```

Reply:
fixed in f05592296ac42d6ff6667e8ed9e238d75e4b83c1